utility/Template/RF24_arch_config.h

```diff
@@ -36,11 +36,10 @@
 #define _BV(x) (1 << (x))
 #define _SPI   spi
 
-#undef SERIAL_DEBUG
-#ifdef SERIAL_DEBUG
```

Comment:
I'm not sure what the intention was here. It doesn't make sense to me. Why un-define a symbol and then immediately check if it is defined? It seems this tactic predates v1.2.0 in 2015 (https://github.com/nRF24/RF24/commit/b320f9e69c130ebe5b75ef15bb6b350fe71a68ea).

Nonetheless, I have removed the `#undef SERIAL_DEBUG` statements because this new name `RF24_DEBUG` should be unique enough to avoid conflicts with other libs.

Reply:
I think it was introduced in BBB support (https://github.com/nRF24/RF24/commit/09269a355759c49854ae844a903ac74143ac7fe8) and then probably copied and pasted multiple times since.